orchestrators/kubernetes/manifests/aqua_csp_009_enforcer/kube_enforcer_advance/003_kube_enforcer_deploy.yaml

```diff
@@ -0,0 +1,101 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: aqua-kube-enforcer
+  namespace: aqua
+spec:
+  ports:
+    - port: 443
+      targetPort: 8443
+      name: envoy
+  selector:
+    app: aqua-kube-enforcer
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aqua-kube-enforcer
+  namespace: aqua
+  labels:
+    app: aqua-kube-enforcer
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: aqua-kube-enforcer
+    spec:
+      serviceAccountName: aqua-kube-enforcer-sa
+      containers:
+        - name: kube-enforcer
+          image: registry.aquasec.com/kube-enforcer:6.0.RC1
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8449
+            - containerPort: 8442
+          env:
+            - name: AQUA_TOKEN
+              value: $token
+            # Specify whether to enable/disable the cache by using "yes", "true", "no", "false" values.
+            # Default value is "yes".
+            - name: AQUA_ENABLE_CACHE
+              value: "yes"
+            # Specify cache expiration period in seconds.
+            # Default value is 60
+            - name: AQUA_CACHE_EXPIRATION_PERIOD
+              value: "60"
+            - name: TLS_SERVER_CERT_FILEPATH
+              value: /certs/server.crt
+            - name: TLS_SERVER_KEY_FILEPATH
+              value: /certs/server.key
+            - name: AQUA_GATEWAY_SECURE_ADDRESS
+              value: aqua-gateway.aqua:8443
+            - name: AQUA_TLS_PORT
+              value: "8449"
+            - name: AQUA_KE_SERVER_PORT
+              value: "8442"
+            - name: CLUSTER_NAME
+              value: $clustername
```

Comment:
should change $clustername no ?

Reply:
Thanks, updated PR with a default value.